### ISSUE TITLE
expo-updates crashes the app in Release build on RN 0.70 bare project

### ISSUE THREAD

**Issue by ziyoshams** [expo/expo#19179]
### Summary

installing `expo-updates` on a bare RN project crashes the app on Release build on iOS. No extra config, just bare minimum installation. 

### What platform(s) does this occur on?

iOS

### Environment

```
  expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 12.4
      Shell: 5.8.1 - /bin/zsh
    Binaries:
      Node: 16.15.1 - /usr/local/bin/node
      Yarn: 1.22.19 - /usr/local/bin/yarn
      npm: 8.11.0 - /usr/local/bin/npm
      Watchman: 2022.05.16.00 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.11.2 - /Users/shamsziyo/.gem/ruby/2.6.0/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    IDEs:
      Android Studio: 2021.1 AI-211.7628.21.2111.8193401
      Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^46.0.0 => 46.0.10
      react: 18.1.0 => 18.1.0
      react-native: 0.70.1 => 0.70.1
    npmGlobalPackages:
      eas-cli: 2.0.0
      expo-cli: 6.0.5
    Expo Workflow: bare
```

### Minimal reproducible example

https://github.com/ziyoshams/RN-70-expo-updates-crash

**Comment by Kudo** [comment]
hi there! that is an issue from install-expo-modules. i will follow up accordingly. in the meantime, please change your AppDelegate.mm as following, that would help to fix your problem.

```diff
--- a/ios/rn70expoupdates/AppDelegate.mm
+++ b/ios/rn70expoupdates/AppDelegate.mm
@@ -44,7 +44,7 @@ static NSString *const kRNConcurrentRoot = @"concurrentRoot";
 #endif

   NSDictionary *initProps = [self prepareInitialProps];
-  UIView *rootView = RCTAppSetupDefaultRootView(bridge, @"rn70expoupdates", initProps);
+  UIView *rootView = [self.reactDelegate createRootViewWithBridge:bridge moduleName:@"rn70expoupdates" initialProperties:initProps];

   if (@available(iOS 13.0, *)) {
     rootView.backgroundColor = [UIColor systemBackgroundColor];
```

**Comment by ziyoshams** [comment]
> hi there! that is an issue from install-expo-modules. i will follow up accordingly. in the meantime, please change your AppDelegate.mm as following, that would help to fix your problem.
> 
> ```diff
> --- a/ios/rn70expoupdates/AppDelegate.mm
> +++ b/ios/rn70expoupdates/AppDelegate.mm
> @@ -44,7 +44,7 @@ static NSString *const kRNConcurrentRoot = @"concurrentRoot";
>  #endif
> 
>    NSDictionary *initProps = [self prepareInitialProps];
> -  UIView *rootView = RCTAppSetupDefaultRootView(bridge, @"rn70expoupdates", initProps);
> +  UIView *rootView = [self.reactDelegate createRootViewWithBridge:bridge moduleName:@"rn70expoupdates" initialProperties:initProps];
> 
>    if (@available(iOS 13.0, *)) {
>      rootView.backgroundColor = [UIColor systemBackgroundColor];
> ```

Worked like a charm. Thanks @Kudo
